src/store/account/account.store.impl.ts

```diff
@@ -66,10 +67,16 @@ export class AccountStoreImpl extends StoreImpl implements AccountStore {
   }
 
   setWeb3User(user: Types.User) {
+    console.log('setting')
       this._web3User = user
    }
 
-  userExists(): boolean {
+  userSignedIn(): boolean {
     return !!this._web3User;
   }
+
+
+  userExists(): boolean {
+    return this.errorCode !== 10;
```

Comment:
Please add a constant on top with the property name defining what the value 10 means!